test/cli.js

```diff
@@ -0,0 +1,107 @@
+var chai = require('chai');
+var child = require('child_process');
+var util = require('./util');
+var exec = child.exec;
+var expect = chai.expect;
+
+describe('cli', function() {
+  var input = util.read('fixtures/cli/input');
+  var output = util.read('fixtures/cli/input.out');
+
+  afterEach(function() {
+    util.remove('fixtures/cli/output');
+  });
+
+  it('should read from a file and write to a file', function(done) {
+    exec('node bin/suitcss -c test/config/noautoprefixer.config.js test/fixtures/cli/input.css test/fixtures/cli/output.css', function(err) {
```

Comment:
yeah I am not suggesting to do this here and now anyway np!
this could be a patch to v3